core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -154,7 +156,7 @@ static void safeClose(Channel ch) {
         }
     }
 
-    private final ServerConfig config;
+    private volatile ServerConfigHolder configHolder;
```

Comment:
Looks like this can be `final`, because `configHolder` is not replaced but only its content (`config`) is.

Reply:
Yes. Will make the change